scripts/update-bindata.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script ensures that enable type directives are not merged to
+# the tree without corresponding fixture to ensure federation of the
+# target type will be tested.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT_DIR="$(cd "$(dirname "$0")/.." ; pwd)"
+
+cd "${ROOT_DIR}"
```

Comment:
My previous comment was a minor question regarding the need to prefix the binary with `./bin/` given that `command` was checking for the binary in the `PATH` (i.e. if the conditional succeeded the binary could be invoked without a path). I wasn't suggesting removing the conditional, since i's nice for the user if the script detects that the command is missing and it can then suggest that the user invoke `scripts/download-binaries.sh`.